src/mcts/params.h

```diff
@@ -53,6 +53,7 @@ class SearchParams {
   float GetCpuctFactor(bool at_root) const {
     return at_root ? kCpuctFactorAtRoot : kCpuctFactor;
   }
+  int GetTwoFoldDrawLevel() const {return options_.Get<int>(kTwoFoldDrawLevelId); }
```

Comment:
clang-format indent style

Reply:
What does this mean, and what is the suggested change?

Reply:
I guess it means that space before return is expected. At least I would expect a space there

Reply:
Added the space resp. moved the return statement to the next line as it exceeded 80 characters.